CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing to `PIXL`.
+
+👍🎉 First off, thanks for taking the time to contribute! 🎉👍
+
+When contributing to this repository, please first discuss the change you wish to make via issue, instant message in clara-agx chatbox, or any other method with the owners of this repository before making a change. 
+Please note we have a [code of conduct](CODE_OF_CONDUCT.md), please follow it in all your interactions with the project.
+
+## :octocat: Setting up project locally. 
+1. Generate your SSH keys as suggested [here](https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
+2. Clone the repository by typing (or copying) the following lines in a terminal
+```
+git clone git@github.com:UCLH-Foundry/PIXL.git
+```
+
+## New issues
+* Open an issue (bug report, feature request, or something is not working): https://github.com/UCLH-Foundry/PIXL/issues/new/choose 
+* Workflow for issue management 
+```mermaid
+  flowchart TD;
+      Z[Bug Reported] -->A[...];  
+      A[Bug resolution] -->B(Testing OK?);
+      B--Yes-->C[Prepare commit];
+      B--No-->D[Reopen issue];
+      D----> A[Bug resolution];
+      C ----> E[Request Review];
+      E ----> F[Commit changes];
+      F --> H[Merge PR and close issue]
+```  
+
+## Committing and pushing changes 
+1. Clone this repo 
+```
+git clone git@github.com:UCLH-Foundry/PIXL.git
+``` 
+2. Create new branch using issue number
+```
+git checkout -b ISSUENUMBER-branch-name 
```

Comment:
Id be tempted to make it more explicit that you replace these. maybe surrounding by `<` or `{` for all of the steps

Reply:
I guess could imagine someone coming the the project who is inexperienced and assuming that if they were working on issue 422 "fix flabber". Then they'd make a branch `422-branch-name`, but if you don't thinks that's likely I'm not too bothered by it

Reply:
Thanks, just added changes here 01c71b12bc564aaf722f52420ce85b35476feeb8